package.json

```diff
@@ -36,5 +36,10 @@
   },
   "resolutions": {
     "usehooks-ts@^2.7.2": "patch:usehooks-ts@npm:^2.7.2#./.yarn/patches/usehooks-ts-npm-2.7.2-fceffe0e43.patch"
+  },
+  "dependencies": {
+    "@metamask/eth-sig-util": "^7.0.0",
+    "eth-crypto": "^2.6.0",
+    "ethers": "5.7.2"
```

Comment:
Pleaase install these within the `packages/nextjs` folder, I'm sure it works this way but it's supposed to be there.